Classes/Converter/FormValueTimestampConverter.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace AOE\SchedulerTimeline\Converter;
+
+/*
+ * (c) 2020 AOE GmbH <dev@aoe.com>
+ *
+ * This file is part of the TYPO3 Scheduler Timeline Extension.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+class FormValueTimestampConverter
+{
+    public static function convertValueToStrToTimeValue(string $value): string
```

Comment:
the class name is speaking enough, perhaps just `"public static function fromString(string ...)"`

Reply:
Can be. I like it this way. because you cannot be in doubt what it does. fromString() I would not know what to expect from the function.